src/test/java/marquez/db/mappers/DatasetRowExtendedMapperTest.java

```diff
@@ -0,0 +1,162 @@
+    assertThat(rowExtended.getName()).isEqualTo(NAME.getValue());
+    assertThat(rowExtended.getUrn()).isEqualTo(URN.getValue());
+    assertThat(rowExtended.getDatasourceUrn()).isEqualTo(DATASOURCE_URN.getValue());
+    assertThat(rowExtended.getDescription()).isEqualTo(DESCRIPTION.getValue());
+    assertThat(rowExtended.getCurrentVersionUuid()).isEqualTo(CURRENT_VERSION_UUID);
+  }
+
+  @Test
+  public void testMap_row_noDescription() throws SQLException {
+    final Object exists = mock(Object.class);
+    final ResultSet results = mock(ResultSet.class);
+    when(results.getObject(Columns.ROW_UUID)).thenReturn(exists);
+    when(results.getObject(Columns.CREATED_AT)).thenReturn(exists);
+    when(results.getObject(Columns.UPDATED_AT)).thenReturn(exists);
+    when(results.getObject(Columns.NAMESPACE_UUID)).thenReturn(exists);
+    when(results.getObject(Columns.DATASOURCE_UUID)).thenReturn(exists);
+    when(results.getObject(Columns.NAME)).thenReturn(exists);
+    when(results.getObject(Columns.URN)).thenReturn(exists);
+    when(results.getObject(Columns.DATASOURCE_URN)).thenReturn(exists);
+    when(results.getObject(Columns.DESCRIPTION)).thenReturn(NO_DESCRIPTION.getValue());
+    when(results.getObject(Columns.CURRENT_VERSION_UUID)).thenReturn(exists);
+
+    when(results.getObject(Columns.ROW_UUID, UUID.class)).thenReturn(ROW_UUID);
+    when(results.getTimestamp(Columns.CREATED_AT)).thenReturn(Timestamp.from(CREATED_AT));
+    when(results.getTimestamp(Columns.UPDATED_AT)).thenReturn(Timestamp.from(UPDATED_AT));
+    when(results.getObject(Columns.NAMESPACE_UUID, UUID.class)).thenReturn(NAMESPACE_ROW_UUID);
+    when(results.getObject(Columns.DATASOURCE_UUID, UUID.class)).thenReturn(DATASOURCE_ROW_UUID);
+    when(results.getString(Columns.NAME)).thenReturn(NAME.getValue());
+    when(results.getString(Columns.URN)).thenReturn(URN.getValue());
+    when(results.getString(Columns.DATASOURCE_URN)).thenReturn(DATASOURCE_URN.getValue());
+    when(results.getObject(Columns.CURRENT_VERSION_UUID, UUID.class))
+        .thenReturn(CURRENT_VERSION_UUID);
+
+    final StatementContext context = mock(StatementContext.class);
+
+    final DatasetRowExtendedMapper rowExtendedMapper = new DatasetRowExtendedMapper();
+    final DatasetRowExtended rowExtended = rowExtendedMapper.map(results, context);
+    assertThat(rowExtended.getUuid()).isEqualTo(ROW_UUID);
+    assertThat(rowExtended.getCreatedAt()).isEqualTo(CREATED_AT);
+    assertThat(rowExtended.getUpdatedAt()).isEqualTo(UPDATED_AT);
+    assertThat(rowExtended.getNamespaceUuid()).isEqualTo(NAMESPACE_ROW_UUID);
+    assertThat(rowExtended.getDatasourceUuid()).isEqualTo(DATASOURCE_ROW_UUID);
+    assertThat(rowExtended.getName()).isEqualTo(NAME.getValue());
+    assertThat(rowExtended.getUrn()).isEqualTo(URN.getValue());
+    assertThat(rowExtended.getDatasourceUrn()).isEqualTo(DATASOURCE_URN.getValue());
+    assertThat(rowExtended.getDescription()).isNull();
```

Comment:
Shouldn't this be `Description.NO_DESCRIPTION.getValue()`?

Reply:
> assertThat(rowExtended.getDescription()).isNull();

Rows don't use Marquez defined types (ex: `Description`). In this case, a description in a row is defined as a string, and would be null. So the `sNull()` is fine, but I see how it can be documented a bit better by using  `NO_DESCRIPTION.getValue()`. I'll make the change 👍